File: Herencia/Ejercicio1/Java/Moto.java
package Herencia.Ejercicio1.Java;

public class Moto extends Vehiculo {
    private String cilindrada, tipo_motor;

    public Moto(String ma, String mo, int a, int p, String c, String t) {
        super(ma, mo, a, p);
        this.cilindrada = c;
        this.tipo_motor = t;
    }

    public void mostrar_info() {
    super.mostrar_info();
    System.out.println("Cilindrada de la moto: "+cilindrada); 
    System.out.println("Tipo de motor: "+tipo_motor);
}
}

File: Herencia/Ejercicio1/python/Moto.py

from Vehiculo import Vehiculo
class Moto(Vehiculo):  
    def __init__(self, marca, modelo, anio, precio_base, cilindrada, tipo_motor):     
        super().__init__(marca, modelo, anio, precio_base)  
        self.cilindrada = cilindrada
        self.tipo_motor = tipo_motor
    
    def mostrar_info(self):
        super().mostrar_info()        
        print(f"Cilindrada de la moto: {self.cilindrada}")        
        print(f"Tipo de motor: {self.tipo_motor}")
